### ISSUE TITLE
Used version of spymemcached.

### ISSUE THREAD

**Issue by fakier073** [magro/memcached-session-manager#349]
In the SetupAndConfiguration wiki it is mentioned to use (in case of memcached as backend for session store) spymemcached 2.11.1.

But that jar is of 2014, meanwhile there were a lot of new releases for spymemcached (2.12.3 as we speak).

Is there a special reason  we should still use version 2.11.1 instead of a newer version?

**Comment by magro** [comment]
Nope, it was just not tested. Does it work for you? Then we can surely update the wiki.

**Comment by fakier073** [comment]
Gonna test a while with a newer version. I will share my findings here then.

**Comment by fakier073** [comment]
Tested a while with newer version. So far, I didn't have any issues.

**Comment by magro** [comment]
Great! Can you update the wiki, and maybe submit a PR with the new spymemcached version?
